napari_assistant/_gui/_button_grid.py

```diff
@@ -14,16 +15,31 @@
     QListWidget::item {
         width: 68;
         height: 85;
-        border-radius: 0;
```

Comment:
I _think_ we may want to keep this...

Reply:
So I tried to keep as many things from the style sheet as possible but this line was somehow blocking that I could set the background color of the buttons. I guess we might need to find an alternative solution but I need to build up my courage to dive deeper into Qt stuff again :D